template/Dockerfile

```diff
@@ -1,16 +1,14 @@
-FROM node:10-alpine
+FROM node:current-alpine
+
+ENV NODE_ENV=production
 
 RUN mkdir /app
 WORKDIR /app
 
+COPY package.json package-lock.json ./
 
-COPY package*.json ./
-
-RUN npm ci 
+RUN npm install --production
 
 COPY . .
 
```

Comment:
`npm run build` is missing.

Reply:
don't use npm install --production in dockerfile?